src/api/worker/search/IndexUtils.js

```diff
@@ -29,6 +29,10 @@ import type {User} from "../../entities/sys/User"
 import type {GroupMembership} from "../../entities/sys/GroupMembership"
 import {TypeRef} from "../../common/utils/TypeRef";
 
+export function encryptAllowedExternalAddress(key: Aes256Key, address: string, dbIv: Uint8Array): Uint8Array {
```

Comment:
It's a bit odd that it's in IndexUtils

Reply:
I guess it was put there a while ago, any idea where a better place would be?